app/assets/scripts/views/ministry.js

```diff
@@ -44,24 +44,16 @@ var Ministry = React.createClass({
     const markers = getProjectCentroids(ministryProjects, get(this.props.api, 'geography.' + GOVERNORATE + '.features'));
     const mapLocation = getFeatureCollection(markers);
 
-    const projectBudgets = ministryProjects
-      .map((project) => project.budget)
-      .reduce((a, b) => a.concat(b), []);
-
     const chartData = ministryProjects.map((project) => {
       return {
         name: project.name,
         link: path.resolve(basepath, 'projects', project.id),
-        value: project.budget.reduce((cur, item) => cur + item.fund.amount, 0)
+        value: project.number_served.number_served
       };
     }).sort((a, b) => b.value > a.value ? -1 : 1);
 
-    // TODO change this to 2 amounts dispursed and remaining
-    const totalBudget = projectBudgets.reduce((currentValue, budget) => {
-      return budget.fund.amount + currentValue;
-    }, 0);
-
     const singleProject = ministryProjects.length <= 1 ? ' funding--single' : '';
+    const activeProjects = ministryProjects.filter((project) => project.actual_end_date).length;
```

Comment:
For consistency's sake, this should be called `numActiveProjects` or be initialized as the array, and then have it's length checked in the template.